subprojects/packagefiles/libarchive/meson.build

```diff
@@ -276,62 +286,86 @@ else
   endif
 endif
 
-if meson.version().version_compare('>= 0.60')
-  iconv_dep = dependency('iconv', disabler: true, required: get_option('iconv'))
+if not get_option('iconv').disabled() and meson.version().version_compare('>= 0.60')
```

Comment:
This is turning `enabled()` into `auto()`, since this will automatically be disabled if the user has an old Meson version. Why don't we just bump the required version to 0.60, that's still pretty old at this point?

Reply:
> This is turning enabled() into auto(), since this will automatically be disabled if the user has an old Meson version.

No, if `-Diconv=enabled` is used and the meson version is too old this will result in an error: `Feature iconv cannot be enabled`, instead of silently disabling iconv support (the previous behavior).

Reply:
Okay, I see where you're handling that now. That seems like an odd place to handle it, instead of here.